continuous_integration/scripts/test_imports.sh

```diff
@@ -25,6 +25,6 @@ test_import () {
 
 test_import ""                                "import dask, dask.base, dask.multiprocessing, dask.threaded, dask.optimization, dask.bag, dask.delayed, dask.graph_manipulation, dask.layers"
 test_import "numpy"                           "import dask.array"
-test_import "pandas"                          "import dask.dataframe"
+test_import "pandas dask-expr"                "import dask.dataframe"
```

Comment:
I think this means `dask-expr` is now a requirement to `import dask.dataframe` (I could be missing something). Is that intentional? If so, we should update requirements too

Reply:
yes, I mentioned this in an earlier offline conversation as well that this is still a todo